amd/src/cardnumber.js

```diff
@@ -0,0 +1,33 @@
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Helper for card number filter
+ *
+ * @module     mod_kanban/cardnumber
+ * @copyright  2024 ISB Bayern
+ * @author     Stefan Hanauska <stefan.hanauska@csg-in.de>
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+export const init = (element) => {
+    document.querySelectorAll('#' + element + ' .mod_kanban_card_number').forEach((el) => {
+        el.addEventListener('click', (event) => {
+            document.querySelector(
+                `.mod_kanban_card[data-number="${event.target.dataset.id}"] .mod_kanban_description_trigger`
```

Comment:
Es wird nicht überprüft, ob es die Karte mit der Nummer gibt. Könnte mittlerweile gelöscht worden sein. 
Ggf. Rückmeldung, dass es die Karte nicht gibt?

Reply:
Es erscheint jetzt ein Modal, das auf das Fehlen der Karte hinweist.

Reply:
Leider funktioniert bei mir (Firefox, Windows) die Verlinkung weiterhin erst nach einem reload. Nun wird ohne reload immer "Card not found" ausgegeben